boot/bootutil/include/bootutil/crypto/ecdsa_p256.h

```diff
@@ -120,46 +136,42 @@ static inline int bootutil_ecdsa_p256_verify(bootutil_ecdsa_p256_context *ctx,
 typedef mbedtls_ecdsa_context bootutil_ecdsa_p256_context;
 static inline void bootutil_ecdsa_p256_init(bootutil_ecdsa_p256_context *ctx)
 {
-    mbedtls_ecdsa_init(ctx);
+	mbedtls_ecdsa_init(ctx);
```

Comment:
There are quite a few weird whitespace changes. Not sure what it is.

Reply:
That's the clang-format that I used I think. If I don't use it then I am getting checkpatch failures in most of the lines of the file. Not 100% sure why to be honest, vscode showed only spaces as indent for this file and checkpatch doesn't like that.

Reply:
The coding style in MCUboot is different from Zephyr-